sliding-sync/default.nix

```
@@ -0,0 +1,117 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.matrix-synapse.sliding-sync;
+in
+{
+  disabledModules = [ "services/matrix/matrix-sliding-sync.nix" ];
+
+  options.services.matrix-synapse.sliding-sync = {
+    enable = lib.mkEnableOption (lib.mdDoc "sliding sync");
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.callPackage ../pkgs/matrix-sliding-sync { };
+      description = "What package to use for the sliding-sync proxy.";
+    };
+
+    enableNginx = lib.mkEnableOption "Should this module autogenerate nginx config";
```

Docstring will be *Whether to enable Should this module autogenerate nginx config*

```suggestion
    enableNginx = lib.mkEnableOption lib.mdDoc("autogenerated nginx config");
```